### ISSUE TITLE
Doesn't work

### ISSUE THREAD

**Issue by Darkhydrawolf** [Annalouu/an-policecarmegaphone#4]
I followed the read me to the letter and it doesn't work

**Comment by JericoFX** [comment]
Do you have the police job?, can you try to give the job again?

**Comment by Darkhydrawolf** [comment]
yes I had the police job and I also followed your directions in your read me to have it work for the FIB in my server still nothing.

**Comment by JericoFX** [comment]
can you tell the error?

**Comment by JericoFX** [comment]
i made a new PR, can you check it?
